invenio/modules/accounts/templates/accounts/settings/helpers_base.html

```diff
@@ -18,10 +18,10 @@
 #}
 
 
-{%- macro panel_start(title, icon="", btn_text='', btn='', btn_icon='', btn_href='', btn_class='btn-default', btn_name='', id="", with_body=True) %}
+{%- macro panel_start(title, icon="", panel_extra_class="", btn_text='', btn='', btn_icon='', btn_href='', btn_class='btn-default', btn_name='', id="", with_body=True) %}
```

Comment:
Sorry what I meant was to add ```panel_extra_class="panel-default"``` here in the macro parameters as default value.

Like this everything will behave in the same way for people using this macro but still it allows to set something different than ```panel-default``` (hardcoded in this version)